Dockerfile

```diff
@@ -24,6 +24,9 @@ ENV CONCOURSE_WORKER_GARDEN_DNS_PROXY_ENABLE  true
 ENV CONCOURSE_WORK_DIR                /worker-state
 ENV CONCOURSE_WORKER_WORK_DIR         /worker-state
 
+# make `ctr` target the default concourse namespace
+ENV CONTAINERD_NAMESPACE=concourse
```

Comment:
this was added to `concourse/dev` more as a way of quickly leveraging `ctr` for debugging without having to type `CONTAINERD_NAMESPACE` or use the `--namespace` flag - given that the `concourse` image generated out of this repo is not for debugging / local dev, wdyt of getting this one removed?

Reply:
Ah I see. Will remove.